cypress/support/commands.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="cypress" />
+
+Cypress.Commands.add("getBySel", (testId) => {
```

Comment:
Please change naming to getByTestId. getBySel is not clear. What Sel presents?

Reply:
Ok. Get by Sel (selector) is a ref to 
![image](https://github.com/MykhailoForge/prep-proj/assets/135103037/bad8060c-e668-4346-8bc7-7494ff98f18c)

Reply:
Sure, But in this example it references to a Selector. Project can feature several custom selectors, so it's better to stick to the context. In this case we have more flexibility.

Reply:
Yeah, sure, obvious naming issue. I just share the link to the doc moment i refered to.